node/src/main/resources/hardcoded-peers.txt

```diff
@@ -0,0 +1,676 @@
+2.39.173.126:8333
```

Comment:
Where are these coming from? It might be nice to add a comment to the top of the file indicating where to get these or how to update these in the future